### PR TITLE
docs: add docstring for tirg image encoder

### PR DESCRIPTION
**Pull request by bwanglzu**
Update docstring for `TirgImageEncoder`.

### CODE REVIEW

File: encoders/image/TirgImageEncoder/__init__.py [hunk 1]
```diff
@@ -11,13 +11,26 @@
 
 
 class TirgImageEncoder(BaseTorchEncoder):
+    """
+    Encode B x (Channel x Height x Width) ndarray into B * d ndarray.
+
+    :class:`TirgImageEncoder` is original proposed in the paper *Composing Text and Image for Image Retrieval - An Empirical Odyssey*.
```
Comment (JoanFM) [inline encoders/image/TirgImageEncoder/__init__.py:17]: is originally

**Review by JoanFM — CHANGES_REQUESTED** [review]
Make sure to update also Readme.md and the manifest properly to point to it

File: encoders/image/TirgImageEncoder/README.md [hunk 2]
```diff
@@ -1,3 +1,64 @@
...
-`TirgImageEncoder` encodes data from a ndarray, potentially B x (Channel x Height x Width) into a B x d ndarray
\ No newline at end of file
+**TIRG** model was original proposed in [Composing Text and Image for Image Retrieval - An Empirical Odyssey](https://arxiv.org/abs/1812.07119).
+It has been used in Jina multimodal retrieval example.
+
+During the indexing time, the **TIRG** model takes `TirgImageEncoder` and encode images into embeddings.
+At query time, the **TIRG** model takes `TirgMultimodalEncoder` and encode images together with it's associated texts into embeddings.
+
+As a part of **TIRG** model,
+`TirgImageEncoder` encodes data from a ndarray, potentially B x (Channel x Height x Width) into a B x d ndarray.
```
Comment (davidbp) [inline encoders/image/TirgImageEncoder/README.md:10]: what is B ? (batch size?) if so it should be defined

File: encoders/image/TirgImageEncoder/__init__.py [hunk 3]
```diff
@@ -11,13 +11,26 @@
 
 
 class TirgImageEncoder(BaseTorchEncoder):
+    """
+    Encode B x (Channel x Height x Width) ndarray into B * d ndarray.
```
Comment (davidbp) [inline encoders/image/TirgImageEncoder/__init__.py:15]: Again, B is not defined

File: encoders/image/TirgImageEncoder/__init__.py [hunk 4]
```diff
@@ -47,15 +61,23 @@ def _get_features(self, data):
     @batching
     @as_ndarray
     def encode(self, data: 'np.ndarray', *args, **kwargs) -> 'np.ndarray':
+        """
+        Encode input data into `np.ndarray`.
+
+        :param data: Image to be encoded, expected a `np.ndarray` of B x (Channel x Height x Width).
```
Comment (davidbp) [inline encoders/image/TirgImageEncoder/__init__.py:67]: B not defined

**Review by davidbp — CHANGES_REQUESTED** [review]
Documentation presents variables such as `B` that are not defined.
`Width` might be inferred from the name, but `B`  is not easily inferred (unless I am missing something and this is a convention that it is used).

File: encoders/image/TirgImageEncoder/README.md (hunk 2 above)
Comment (bwanglzu) [reply encoders/image/TirgImageEncoder/README.md:10]: got it, thanks!

File: encoders/image/TirgImageEncoder/__init__.py (hunk 1 above)
Comment (davidbp) [reply encoders/image/TirgImageEncoder/__init__.py]: `ndarray` ->`np.ndarray`

**Review by davidbp — APPROVED** [review]

**Review by davidbp — DISMISSED** [review]

File: encoders/image/TirgImageEncoder/README.md [hunk 5]
```diff
@@ -1,3 +1,97 @@
...
-`TirgImageEncoder` encodes data from a ndarray, potentially B x (Channel x Height x Width) into a B x d ndarray
\ No newline at end of file
+**TIRG** model was original proposed in [Composing Text and Image for Image Retrieval - An Empirical Odyssey](https://arxiv.org/abs/1812.07119).
+It has been used in Jina multimodal retrieval example.
+The **TIRG** model uses `TirgImageEncoder` and encode images into embeddings.
+
+As a part of **TIRG** model,
+`TirgImageEncoder` encodes data from a ndarray, potentially BatchSize x (Channel x Height x Width) into a BatchSize x d ndarray.
+
+Note: At query time, the **TIRG** model takes `TirgMultimodalEncoder` and encode images together with it's associated texts into embeddings.
```
Comment (JoanFM) [inline encoders/image/TirgImageEncoder/README.md:10]: I would say, if you want to use it as in the `example` or as in the paper

Comment (bwanglzu) [reply encoders/image/TirgImageEncoder/README.md]: updated

**Review by JoanFM — APPROVED** [review]

**Pull request merged**